packages/integration-objects/srv/validations/ValidatePattern.js

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+const { ValidationError } = require('../exceptions/validation-error');
+const { logger } = require('../logging/Logger');
+const ErrorStatuses = require('../utils/ErrorStatuses');
+
+async function validatePattern(patternType, fieldName, fieldValue, maxLength) {
+  if (!patternType.test(fieldValue)) {
+    logger.error(`Provide a valid value for ${fieldName}. The value must be 1 - ${maxLength} characters long.`);
```

Comment:
Ok still I would prefer a string like `humanUnderstandablePattern` which is more or less the regex you are testing so that a human can understand it - could be optional. Because the pattern you give in could be anything not necessarily a regex checking the length right? But up to you.